packages/assistant-stream/src/core/serialization/PlainText.ts

```
@@ -27,16 +27,13 @@ export class PlainTextEncoder
-              throw new Error(
-                `tool-call-begin and tool-call-delta are not supported in plain text serialization`,
-              );
-
             default:
-              const exhaustiveCheck: never = type;
-              throw new Error(`unsupported chunk type: ${exhaustiveCheck}`);
+              const unsupportedType:
+                | "tool-call-begin"
+                | "tool-call-delta"
+                | "tool-result"
+                | "error" = type;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)